src/common/snippets/include/snippets/lowered/loop_manager.hpp

```diff
@@ -154,56 +214,139 @@ class LinearIR::LoopManager {
                            const std::vector<T>& target_ports, bool is_entry = true) {
         for (auto loop_id : loop_ids) {
             update_loop_port(loop_id, actual_port, target_ports, is_entry);
         }
     }
-    // Sort Loop Ports by expression locations in Linear IR
+    /**
+     * @brief Sort Loop Ports by expression locations in Linear IR
+     * @param loop_begin_pos the first expression iterator of the Loop
+     * @param loop_end_pos the next iterator after the last expression
+     * @param loop_id target Loop ID
+     */
     void sort_loop_ports(LinearIR::constExprIt& loop_begin_pos, LinearIR::constExprIt& loop_end_pos, size_t loop_id);
-
-    // When the previous expression was replaced with new expressions (decomposition), the method updates the corresponding Loop.
-    // If ports of decomposed expression were the Loop ports, these Loop ports may be updated by parameters `entries` and `exits`
-    // Note: This method should be removed when Softmax decomposition will be moved on data flow pipeline since
-    //       all decompositions should be call on this pipeline
+    /**
+     * @brief When the previous expression was replaced with new expressions (decomposition), the method updates the corresponding Loop.
+     *        If ports of decomposed expression were the Loop ports, these Loop ports may be updated by parameters `entries` and `exits`
+     *        Note: This method should be removed when Softmax decomposition will be moved on data flow pipeline since
+     *              all decompositions should be call on this pipeline
+     * @param new_expr_begin the first expression iterator
+     * @param new_expr_end the next iterator after the last expression
+     * @param decomposed_expr the expression that is decomposed into several other exprs
+     * @param loop_id the new expression will be marked by this loop ID
+     * @param new_entries new input loop ports
+     * @param new_exits new output loop ports
+     */
     void expression_replacement(constExprIt new_expr_begin, constExprIt new_expr_end, const ExpressionPtr& decomposed_expr,
                                 size_t loop_id, const std::vector<ExpressionPort>& new_entries, const std::vector<ExpressionPort>& exits);
+    /**
+     * @brief Find iterators of first entry loop point and last exit point (bounds of Loop) when explicit Loop exprs haven't been inserted in LinearIR
+     *        Note: This method cannot be called when explicit loop exprs are inserted since Loop bounds are defined by LoopBegin and LoopEnd after the insertion!
+     * @param linear_ir linear IR
+     * @param loop_id target Loop ID
+     * @param loop_begin_pos the ref of first expression iterator of the Loop
+     * @param loop_end_pos the ref of next iterator after the last expression
+     */
+    void get_loop_bounds(const LinearIR& linear_ir, size_t loop_id, LinearIR::constExprIt& loop_begin_pos, LinearIR::constExprIt& loop_end_pos) const;
```

Comment:
Usually when you see a `get_something(...)` method, you expect that it returns `something`. So I would suggest to change the signature a little bit:
```suggestion
    std::pair<LinearIR::constExprIt, LinearIR::constExprIt> get_loop_bounds(const LinearIR& linear_ir, size_t loop_id) const;
```
What do you think?